app/views/templates/run_command.erb

```diff
@@ -0,0 +1,13 @@
+<%#
+kind: job_template
+name: Run Command - SSH Default
+job_name: RunCommand
+provider_type: Ssh
+template_inputs:
+- name: command
+  description: Command to run on the host
+  input_type: user
+  required: true
+%>
+
```

Comment:
It's in my code, I fixed the regex.  I remove the metadata at the top. Foreman leaves it which imho many users find confusing, and our metadata is 3-4x longer.